### PR TITLE
Add/update Offscreen Documents entries in What's New/Known Issues

### PR DESCRIPTION
**Pull request by oliverdunk**
Fixes the Chrome version mentioned in Known Issues, and adds a What's New entry. I've intentionally back-dated the What's New since it's been out for a while, and I also wanted to line it up with the blog post.

### CODE REVIEW

**Review by AmySteam — APPROVED** [review]
LGTM! Thanks @oliverdunk 💯

File: site/en/docs/extensions/whatsnew/index.md [hunk 1]
```diff
@@ -22,6 +22,12 @@ and related policy or other changes. You'll find other notices posted on the [Ex
 
 From Chrome 112, the quota for the [`storage.session`](/docs/extensions/reference/storage/#property-session) property has been increased to approximately 10 MB. This was agreed in the Web Extensions Community Group: [https://github.com/w3c/webextensions/issues/350](https://github.com/w3c/webextensions/issues/350)
 
+### Chrome 109: Offscreen documents {: #m109-offscreen-docs}
+
+<p class="color-secondary-text type--caption">Posted on <time>January 25, 2023</time></p>
+
+Offscreen documents are now available on the stable channel. These provide support for DOM-related features and APIs in Manifest V3. For more information, [read the blog post](/blog/Offscreen-Documents-in-Manifest-v3/).
```
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:29]:
```suggestion
Offscreen documents are now available in extensions. These provide support for DOM-related features and APIs in Manifest V3. For more information, [read the blog post](/blog/Offscreen-Documents-in-Manifest-v3/).
```

Comment (jpmedley) [reply site/en/docs/extensions/whatsnew/index.md:29]: Generally, we probably don't need to say channel unless we want to call out a specific channel earlier than stable.

Can you please add something about this helping transition from background pages to service workers, something like what the first line of Ian's article says.

**Review by jpmedley — CHANGES_REQUESTED** [review]

**Review by jpmedley — APPROVED** [review]
@oliverdunk, Please forgive me. I'm going to merge this. I suddenly have more for 'What's New' than I know what to do with. I need to get the next item queued for Monday.

**Pull request merged**